src/js/store/createStore.js

```diff
@@ -0,0 +1,31 @@
+export default (reducer) => {
```

Comment:
```suggestion
export default function createStore(reducer) {
```
이렇게 표현하면 나중에 파일을 탐색할 때 더 유리하답니다!